client/layout/sidebar-v2/index.tsx

```diff
@@ -0,0 +1,18 @@
+export * from './header';
+export * from './main';
+export * from './footer';
+
+import classNames from 'classnames';
+
+import './style.scss';
+
+type Props = {
+	className?: string;
+	children: React.ReactNode;
+};
+
+const SidebarV2 = ( { children, className = '' }: Props ) => {
```

Comment:
[NITPICK] The `className` doesn't have to have ' ' since classNames will take care of undefined values.